Cargo.toml

```diff
@@ -39,6 +39,13 @@ atomic-polyfill = { version = "0.1.4", optional = true }
 [target.riscv32imc-unknown-none-elf.dependencies]
 atomic-polyfill = { version = "0.1.4", optional = true }
 
+[target.avr-atmega328p.dependencies]
```

Comment:
these targets don't appear in the output of `rustc --print target-list`. the target should be `avr-unknown-gnu-atmega328`

Reply:
If I make the change you suggest, I get the following error:

```
error[E0432]: unresolved import `atomic_polyfill`
  --> /home/thoth/src/heapless/src/pool/singleton/arc.rs:85:5
   |
85 | use atomic_polyfill::{AtomicUsize, Ordering};
   |     ^^^^^^^^^^^^^^^ use of undeclared crate or module `atomic_polyfill`

error[E0432]: unresolved import `atomic_polyfill`
 --> /home/thoth/src/heapless/src/pool/llsc.rs:7:5
  |
7 | use atomic_polyfill::{AtomicPtr, Ordering};
  |     ^^^^^^^^^^^^^^^ use of undeclared crate or module `atomic_polyfill`

error[E0433]: failed to resolve: use of undeclared crate or module `atomic_polyfill`
  --> /home/thoth/src/heapless/src/mpmc.rs:97:25
   |
97 | type AtomicTargetSize = atomic_polyfill::AtomicU8;
   |                         ^^^^^^^^^^^^^^^ use of undeclared crate or module `atomic_polyfill`

error[E0433]: failed to resolve: use of undeclared crate or module `atomic_polyfill`
   --> /home/thoth/src/heapless/src/mpmc.rs:102:17
    |
102 | type Ordering = atomic_polyfill::Ordering;
    |                 ^^^^^^^^^^^^^^^ use of undeclared crate or module `atomic_polyfill`

error[E0392]: parameter `T` is never used
   --> /home/thoth/src/heapless/src/pool/mod.rs:263:17
    |
263 | pub struct Pool<T> {
    |                 ^ unused parameter
    |
    = help: consider removing `T`, referring to it in a field, or using a marker such as `PhantomData`
    = help: if you intended `T` to be a const parameter, use `const T: usize` instead
```